integrations/gitlab/gitlab_integration/utils.py

```diff
@@ -38,6 +41,17 @@ def get_cached_all_services() -> List[GitlabService]:
         return get_all_services()
 
 
+def merge_all_groups(
+    token_group_override_hooks_mapping: TokenWebhookMapping,
+) -> dict[str, WebhookGroup]:
+    all_groups: dict[str, WebhookGroup] = {}
+
+    for webhook_token_obj in token_group_override_hooks_mapping.tokens.values():
+        all_groups.update(webhook_token_obj.groups)
+
+    return all_groups
+
+
```

Comment:
sounds like a redundant util function, its only used in one place, this is also inconsistent with `validate_hooks_tokens_are_in_token_mapping` which is a helper function in the setup file

Reply:
done